quantities/astronomy.hpp

```diff
@@ -15,25 +15,42 @@ namespace astronomy {
 
 // Résolution B2 "Re-définition de l’unité astronomique de longueur" adopted
 // at the XXVIIIth General Assembly of the IAU in 2012.
-constexpr Length AstronomicalUnit      = 149597870700 * si::Metre;
+constexpr Length AstronomicalUnit = 149597870700 * si::Metre;
 
 // See note 4 of Résolution B2 "Sur la recommandation du "point zéro" des
 // échelles de magnitude bolométrique absolue et apparente" adopted at the
 // XXIXth General Assembly of the IAU in 2015.
-constexpr Length Parsec                = 648000 / π * AstronomicalUnit;
+constexpr Length Parsec = 648000 / π * AstronomicalUnit;
 
 // System of nominal solar and planetary conversion constants, Résolution B3
 // "Sur les valeurs  recommandées de constantes de conversion pour une sélection
 // de propriétés solaires et planétaires" adopted at the XXIXth General Assembly
 // of the IAU in 2015.
-constexpr Mass   SolarMass             = 1.98855e30 * si::Kilogram;
-constexpr Mass   JupiterMass           = 1.8986e27 * si::Kilogram;
-constexpr Mass   EarthMass             = 5.9742e24 * si::Kilogram;
-constexpr Length EarthEquatorialRadius = 6.3781e6 * si::Metre;
-constexpr Length SolarRadius           = 6.957e8 * si::Metre;
-
-constexpr Time   JulianYear            = 365.25 * si::Day;
-constexpr Length LightYear             = constants::SpeedOfLight * JulianYear;
+
+// Solar conversion constants.
+constexpr Length      SolarRadius               = 6.957e8 * si::Metre;
+constexpr Irradiance  TotalSolarIrradiance      = 1361 * (si::Watt /
+                                                          Pow<2>(si::Metre));
+constexpr Power       SolarLuminosity           = 3.828e26 * si::Watt;
+constexpr Temperature SolarEffectiveTemperature = 5772 * si::Kelvin;
+constexpr GravitationalParameter SolarGravitationalParameter =
+    1.327'124'4e20 * (Pow<3>(Metre) / Pow<2>(Second));
+
+// Planetary conversion constants.
+// “If equatorial vs. polar radius is not explicitly specified, it should be
+// understood that nominal terrestrial [or jovian] radius refers specifically to
+// [the nominal equatorial radius], following common usage.”
+constexpr Length TerrestrialEquatorialRadius = 6.3781e6 * si::Metre;
+constexpr Length TerrestrialPolarRadius      = 6.3568e6 * si::Metre;
+constexpr Length JovianEquatorialRadius      = 7.1492e7 * si::Metre;
+constexpr Length JovianPolarRadius           = 6.6854e7 * si::Metre;
+constexpr GravitationalParameter TerrestrialGravitationalParameter =
+    3.986'004e14 * (Pow<3>(si::Metre) / Pow<2>(si::Second));
+constexpr GravitationalParameter JovianGravitationalParameter      =
+    1.266'865'3e17 * (Pow<3>(si::Metre) / Pow<2>(si::Second));
```

Comment:
I would have grouped all the parameters for a planet together: equatorial radius/polar radius/gravitational parameter separated by a blank line.  I would also have alphabetized the planets (thus Jovian < Terrestrial).

Reply:
No, let's keep the order used by the IAU tables.